transmart_loader/transmart.py

```diff
@@ -225,10 +227,13 @@ class Study:
     def __init__(self,
                  study_id: str,
                  name: str,
-                 metadata: Optional[StudyMetadata] = None):
+                 metadata: Optional[Dict[str, Dict[str, Any]]] = None):
         self.study_id = study_id
         self.name = name
-        self.metadata = metadata
+        self.metadata = None
+        if metadata and metadata.get('conceptCodeToVariableMetadata'):
+            self.metadata = StudyMetadata(
```

Comment:
We could use pydantic to specify the types of StudyMetadata and VariableMetadata to have the validation automatically.

Reply:
I'll change it